lib/block_orthogonalize.cu

```diff
@@ -15,7 +15,10 @@
 #include <launch_kernel.cuh>
 
 #include <jitify_helper.cuh>
+
+#ifdef GPU_MULTIGRID
```

Comment:
Just curious: why bother adding `#ifdef GPU_MULTIGRID` around the kernel include files?  They won't be compiled if GPU_MULTIGRID isn't enabled anyway since the kernels won't be instantiated so the additional compilation time should be trivial.  And long term, I hope we can do away with as many `#ifdef` conditional as possible.